### DISCUSSION TITLE
(Crypto) Scams on GitHub have gotten out of hand

### DISCUSSION THREAD

**Discussion by TechStudent10** [community/community#83803] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello esteemed users of GitHub.

I've been mentioned in two issues (on repositories I've had no assosiation with) that turn out to be Binance crypto scams

_The issues in question_
![image](https://github.com/community/community/assets/76978184/d63f8271-4a4b-4493-ade6-a79e18c5d837)

The "White Window" issue appears to have had the scam be posted twice.

This needs to be stopped, although it might be a minor annoyance for people who are more tech-literate, for someone who created a GitHub account for the sole reason of creating an issue or just having one for the sake of having one (I've met quite a few people like this), of whom may not be as knowledgable about these scams, are more prone to falling for them.

I've noticed something like this happening a couple months ago where new users were spam pinging random accounts. A GitHub employee assured me that they would take action, but it appears that those actions have failed. (#42869)

Please fix.

_Edit 1:_ I'd also like to note the amount of unrelated discussions being started here, from random users most likely on the wrong repo to outright scams being present here, it seems that GitHub isn't moderating content being posted on their site very well.

**Comment by Akash1134** [comment]
Hi @TechStudent10 , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- Reporting abuse or spam
- Reporting a user
- Reporting an issue or pull request
- Reporting a comment

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Reply by clouedoc** [reply]
How can we report the content? It gets removed before I come online.
The point is that the emails get sent through GitHub, so that they bypass spam filters.
Here's how it looks:

<img width="443" alt="image" src="https://github.com/community/community/assets/13921610/d4f41d01-7017-4f32-b158-b05b4edc0c6e">
<img width="496" alt="image" src="https://github.com/community/community/assets/13921610/09573925-b62d-4743-856a-83d20a310aa3">

**Reply by elvisisvan** [reply]
this is absolutely outrageous, somebody gotta do something about it

**Reply by feynon** [reply]
Getting bombed with them

**Reply by theodesp** [reply]
I got 50 emails today. It's insane!

**Reply by Dante690** [reply]
i got too what do the support team with this..?

**Reply by yikerman** [reply]
Scammers delete their account immediately after spamming people. How is GitHub able to mod this?

**Comment by Murodxojayev1ch** [comment]
Response #1

**Comment by drozdowsky** [comment]
Also affects me, I have to disable email on mentions unfortunately

**Comment by imtianx** [comment]
I also have the problem. How can I block the scam user ？

**Comment by nmenezes1** [comment]
That is true, this must be looked into.

NM,
www.sqlops.com

**Comment by elvisisvan** [comment]
## this mf is one of the culprits, report him, report them all
![Screenshot_20240427-130350](https://github.com/community/community/assets/69122770/581cd912-577e-438d-b702-d47db4b04768)
![Screenshot_20240427-130634](https://github.com/community/community/assets/69122770/8baad8c9-d301-44b6-b4fa-a415a2b2e84a)

**Reply by bijoy26** [reply]
Already started receiving unsolicited mentions for this scam campaign from like issues from 10 different repos 😖

**Reply by smithlarissa176-lang** [reply]
How to Recover Stolen Cryptocurrency: A Guide With Infinite Digital Recovery

Losing cryptocurrency to a scam, hack, or fraudulent investment is devastating. Because blockchain transactions are irreversible, many victims don’t know where to begin—or which companies they can trust. While no recovery method can guarantee results, taking the right steps early can significantly increase the chances of tracing your assets, documenting evidence, and pursuing legitimate recovery channels.

This guide explains how crypto theft happens, what immediate actions to take, and how Infinite Digital Recovery supports victims through transparent, professional, and ethical recovery assistance.

What Happens When Cryptocurrency Is Stolen?
Cryptocurrency theft usually involves one of the following:
	•	Investment scams and fraudulent trading platforms
	•	Phishing attacks that steal seed phrases or logins
	•	Fake customer-service impersonators
	•	Romance investment scams (“pig butchering”)
	•	Compromised wallets or exchange accounts

Immediate Steps to Take After Your Crypto Is Stolen
1. Secure Your Accounts and Devices
Change passwords, enable 2FA, and remove suspicious browser extensions or apps.
2. Gather Evidence
Collect:
	•	Transaction IDs (TXIDs)
	•	Wallet addresses involved
	•	Screenshots of conversations
	•	Platform URLs
	•	Payment confirmations

This evidence is essential for blockchain tracing.
3. Report the Incident
Notify:
	 • Infinite Digital Recovery Via Email:   infinitedigitalrecovery@techie.com 
	• Global reporting channels (IC3, Action Fraud, etc.)
	• The platform where the scammer contacted you (e.g., WhatsApp, Telegram, social media)

Reporting builds a trail that may assist future investigations.

4. Contact a Legitimate Crypto Recovery Professional such as 
Email:  infinitedigitalrecovery@techie.com 
WHATSAPP:   + 132*355*43592

Many “recovery agents” online are fake. Avoid anyone who:
	•	Charges upfront fees for “unlock codes.”
	•	Claims they can “hack wallets.”
	•	Asks for your seed phrase

How Infinite Digital Recovery Helps Crypto Theft Victims

Infinite Digital Recovery provides professional, research-based, and transparent assistance to victims of crypto scams. Rather than promising guaranteed returns—which is impossible in legitimate recovery—they focus on:

✔ 1. Blockchain Forensic Analysis
Using advanced tracing tools, analysts help track the movement of stolen crypto across:
	•	Wallets
	•	Exchanges
	•	Blockchains
	•	Mixers
	•	Cross-chain platforms

This creates a detailed activity report.

✔ 2. Evidence Documentation
Victims receive properly organized case files containing:
	•	Transaction mapping
	•	Suspect wallet identification
	•	Timeline of events
	•	Exchange contact points
	•	Recommended next steps

These files are essential for law enforcement or legal action.

✔ 3. Scam Awareness & Prevention
They also educate victims about:
	•	Red flags
	•	Common recovery scams
	•	Safe ways to store and move crypto
	•	How to avoid being targeted again

Many victims fall for a second scam when searching for help. Scammers often:
	•	Find victims on Reddit, Google, and Facebook
	•	Pretend to be investigators
	•	Demand upfront charges for “unlocking fees,” “gas fees,” or “private key recovery.

Infinite Digital Recovery emphasizes honesty and transparency—no unrealistic claims, no misleading tactics.

Final Thoughts

Recovering stolen cryptocurrency is challenging, but you are not powerless. By taking the right steps early, gathering strong evidence, and working with legitimate professionals, you can improve your chances of tracing your assets and building a solid case.

Infinite Digital Recovery provides victims with honest guidance, blockchain analysis, and professional support—without false promises or risky tactics. Reach out to Infinite Digital Recovery to start your recovery process.

Email:  infinitedigitalrecovery@techie.com 
WHATSAPP:   + 132*355*43592

**Comment by imtianx** [comment]
Hi,pls focus on this issue.

**Reply by imtianx** [reply]
Today I've been tagged dozens of times in 9 repos😖
![WX20240427-170658@2x](https://github.com/community/community/assets/13366797/6708fd6d-aecd-48b4-b867-f12593de8ee9)

**Reply by ettaboyle** [reply]
Hi there,

I have noticed a pattern of users tagging `jmeridth` & `Iron-Ham` -- neither of these people work on the team that manages our community discussions. Can you share any info on why you've tagged them?

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.

**Comment by eigenpaul** [comment]
This is still a pressing issue. Today alone I've been tagged in more than 4 related crypto scam posts

**Comment by x-magic** [comment]
Same thing here on a newer issue from an empty repo by a non-reputable user: 
![image](https://github.com/community/community/assets/3180639/696fb185-d182-415c-8dab-0d4bc3608158)

This is the first time I know that GitHub doesn't have a report feature on repos, issues or replies. As far as I can find there's only an option to report specific user. 

Given the fact that most email services won't sent GitHub emails to spam, these scams/spams are quite effective in regard to penetrability 😏

**Reply by captn3m0** [reply]
There are 21M pull requests and 6.4M issues created by this "Blastoff.zone: Earn 1 ETH" spam. (Writing the title here to keep it searchable).

https://github.com/search?q=%22blastoff.zone%3A+Earn+1+ETH%22&type=issues

**Reply by MHIII** [reply]
Hi when I go to the official bitcont website link from coinbase and look at cloud mining the links provided for cloud mining all revert to nicehash ? do you know if nicehash is a real cloud mining site or is it a scam? Please and thanks!

**Reply by MHIII** [reply]
never mind I just looked at trust pilot and they are a scam company! Thanks

**Comment by KaganCanSit** [comment]
Hello,

Lately I've been receiving emails from Github tagging issues and PR requests opened through various projects. I am aware that these are a phishing attack. But it's getting annoying.

As a society, it would be beneficial for everyone to be informed. We can be absent-minded. Please do not forget;

An organization like Github will not send you this type of email! Be careful when granting permissions to your Github account and the bots, tools, and other content you want to add. Check out the tools.

How can we as a community help you further @github?

Additionally, efforts were made to fulfill the same issue and PR demands through my projects. Is there a way to temporarily turn off the notification email for my own repos?

**Reply by theodesp** [reply]
I can confirm. I got 50 emails overnight from spam comments. My inbox email is full.

**Reply by KaganCanSit** [reply]
I think what should be here is to review requests such as issues and PR. Also, plugin stores need to be tightened up. When it is done from my open source projects, the e-mail is sent through my project. That's when I get very uncomfortable. I should be able to intervene.

**Reply by borutaralf5436-pixel** [reply]
GREAT WHIP RECOVERY CYBER SERVICES TRUSTED EXPERTS IN ONLINE RECOVERY SOLUTIONS PHONE CALL:+1(406)2729101

I was unfortunately deceived and scammed out of $88,000 by someone I trusted to manage my funds during a transaction we carried out together. The experience left me deeply disappointed and hurt, realizing that someone could betray that level of trust without any remorse. Determined to seek justice and recover what was stolen, I began searching for legal assistance and came across numerous testimonials about GREAT WHIP RECOVERY CYBER SERVICES, a group known for helping victims recover lost funds. From what I learned, they have successfully assisted many people facing similar situations, returning stolen funds to their rightful owners in a remarkably short time. In my case, the GREAT WHIP RECOVERY CYBER SERVICES were able to recover my funds within just 48 hours, which was truly unbelievable. Even more reassuring was the fact that the scammer was identified, located, and eventually arrested by local authorities in his region. That outcome brought a great sense of relief and closure. I hope this information helps others who have lost their hard-earned money due to misplaced trust. If you’re in a similar situation, you can contact them through their info below to seek help in recovering your stolen funds. 

Email: greatwhiprecoverycyberservices@gmail.com 
Website https://greatwhiprecoveryc.wixsite.com/greatwhip-site 
Phone Call:+1(406)2729101

**Comment by HappyFrank01** [comment]
I’m pleased to recommend the professional fund recovery service (wisetechhackⓐgmail .com) They has the best and reliable fund recovery agency to help you get back your lost money, stolen crypto coins back. They are the real deal as I was able to recover my stolen $350,000 dollars worth of USDT back after days of being depressed. This team brought back my lost hope, they are the best

**Comment by smithlarissa176-lang** [comment]
How to Recover Stolen Cryptocurrency: A Guide With Infinite Digital Recovery

Losing cryptocurrency to a scam, hack, or fraudulent investment is devastating. Because blockchain transactions are irreversible, many victims don’t know where to begin—or which companies they can trust. While no recovery method can guarantee results, taking the right steps early can significantly increase the chances of tracing your assets, documenting evidence, and pursuing legitimate recovery channels.

This guide explains how crypto theft happens, what immediate actions to take, and how Infinite Digital Recovery supports victims through transparent, professional, and ethical recovery assistance.

What Happens When Cryptocurrency Is Stolen?
Cryptocurrency theft usually involves one of the following:
	•	Investment scams and fraudulent trading platforms
	•	Phishing attacks that steal seed phrases or logins
	•	Fake customer-service impersonators
	•	Romance investment scams (“pig butchering”)
	•	Compromised wallets or exchange accounts

Immediate Steps to Take After Your Crypto Is Stolen
1. Secure Your Accounts and Devices
Change passwords, enable 2FA, and remove suspicious browser extensions or apps.
2. Gather Evidence
Collect:
	•	Transaction IDs (TXIDs)
	•	Wallet addresses involved
	•	Screenshots of conversations
	•	Platform URLs
	•	Payment confirmations

This evidence is essential for blockchain tracing.
3. Report the Incident
Notify:
	 • Infinite Digital Recovery Via Email:   infinitedigitalrecovery@techie.com 
	• Global reporting channels (IC3, Action Fraud, etc.)
	• The platform where the scammer contacted you (e.g., WhatsApp, Telegram, social media)

Reporting builds a trail that may assist future investigations.

4. Contact a Legitimate Crypto Recovery Professional such as 
Email:  infinitedigitalrecovery@techie.com 
WHATSAPP:   + 132*355*43592

Many “recovery agents” online are fake. Avoid anyone who:
	•	Charges upfront fees for “unlock codes.”
	•	Claims they can “hack wallets.”
	•	Asks for your seed phrase

How Infinite Digital Recovery Helps Crypto Theft Victims

Infinite Digital Recovery provides professional, research-based, and transparent assistance to victims of crypto scams. Rather than promising guaranteed returns—which is impossible in legitimate recovery—they focus on:

✔ 1. Blockchain Forensic Analysis
Using advanced tracing tools, analysts help track the movement of stolen crypto across:
	•	Wallets
	•	Exchanges
	•	Blockchains
	•	Mixers
	•	Cross-chain platforms

This creates a detailed activity report.

✔ 2. Evidence Documentation
Victims receive properly organized case files containing:
	•	Transaction mapping
	•	Suspect wallet identification
	•	Timeline of events
	•	Exchange contact points
	•	Recommended next steps

These files are essential for law enforcement or legal action.

✔ 3. Scam Awareness & Prevention
They also educate victims about:
	•	Red flags
	•	Common recovery scams
	•	Safe ways to store and move crypto
	•	How to avoid being targeted again

Many victims fall for a second scam when searching for help. Scammers often:
	•	Find victims on Reddit, Google, and Facebook
	•	Pretend to be investigators
	•	Demand upfront charges for “unlocking fees,” “gas fees,” or “private key recovery.

Infinite Digital Recovery emphasizes honesty and transparency—no unrealistic claims, no misleading tactics.

Final Thoughts

Recovering stolen cryptocurrency is challenging, but you are not powerless. By taking the right steps early, gathering strong evidence, and working with legitimate professionals, you can improve your chances of tracing your assets and building a solid case.

Infinite Digital Recovery provides victims with honest guidance, blockchain analysis, and professional support—without false promises or risky tactics. Reach out to Infinite Digital Recovery to start your recovery process.

Email:  infinitedigitalrecovery@techie.com 
WHATSAPP:   + 132*355*43592

**Comment by richardlouis3201-create** [comment]
THE MOST CREDIBLE CRYPTO RECOVERY: TOP RECOVERY EXPERT

TOP RECOVERY EXPERT is a reliable and legitimate company that can help recover lost cryptocurrency assets. After weeks of wondering if my lost BTC could ever be restored, I realized how frequent cryptocurrency scams have become. When dealing with individuals online, especially regarding money, caution is essential. Recovering stolen cryptocurrency is possible, but it’s important not to fall victim to another scam—there are many fake “recovery companies” worldwide. I personally experienced multiple scams while desperately seeking help to recover my lost funds. Finally, a friend introduced me to TOP RECOVERY EXPERT, a trustworthy and discreet team. They handle everything from securing personal or company websites to recovering cryptocurrency assets. With their help, I successfully recovered $680,000 worth of USDT in just over a week. Their professionalism, discretion, and prompt service were outstanding. If you’ve been compromised, don’t lose hope—and be careful of fraudsters posing as saviors. TOP RECOVERY EXPERT are real professionals in crypto recovery. I am living proof of their effectiveness. you can reach them by email: Toprecoveryexpert@consultant.com OR you contact their Phone Call/Text: +1 (346) 980-9102

**Comment by richardlouis3201-create** [comment]
THE MOST CREDIBLE CRYPTO RECOVERY: TOP RECOVERY EXPERT

TOP RECOVERY EXPERT is a reliable and legitimate company that can help recover lost cryptocurrency assets. After weeks of wondering if my lost BTC could ever be restored, I realized how frequent cryptocurrency scams have become. When dealing with individuals online, especially regarding money, caution is essential. Recovering stolen cryptocurrency is possible, but it’s important not to fall victim to another scam—there are many fake “recovery companies” worldwide. Real hackers work discreetly and do not advertise themselves in such obvious ways. I personally experienced multiple scams while desperately seeking help to recover my lost funds. Finally, a friend introduced me to TOP RECOVERY EXPERT, a trustworthy and discreet team. They handle everything from securing personal or company websites to recovering cryptocurrency assets. With their help, I successfully recovered $680,000 worth of USDT in just over a week. Their professionalism, discretion, and prompt service were outstanding. If you’ve been compromised, don’t lose hope—and be careful of fraudsters posing as saviors. TOP RECOVERY EXPERT are real professionals in crypto recovery. I am living proof of their effectiveness. you can reach them by email: Toprecoveryexpert@consultant.com OR you contact their Phone Call/Text: +1 (346) 980-9102
you can visit website:https://toprecoveryexpert2.wixsite.com/consultant

**Comment by borutaralf5436-pixel** [comment]
GREAT WHIP RECOVERY CYBER SERVICES TRUSTED EXPERTS IN ONLINE RECOVERY SOLUTIONS PHONE CALL:+1(406)2729101

I was unfortunately deceived and scammed out of $88,000 by someone I trusted to manage my funds during a transaction we carried out together. The experience left me deeply disappointed and hurt, realizing that someone could betray that level of trust without any remorse. Determined to seek justice and recover what was stolen, I began searching for legal assistance and came across numerous testimonials about GREAT WHIP RECOVERY CYBER SERVICES, a group known for helping victims recover lost funds. From what I learned, they have successfully assisted many people facing similar situations, returning stolen funds to their rightful owners in a remarkably short time. In my case, the GREAT WHIP RECOVERY CYBER SERVICES were able to recover my funds within just 48 hours, which was truly unbelievable. Even more reassuring was the fact that the scammer was identified, located, and eventually arrested by local authorities in his region. That outcome brought a great sense of relief and closure. I hope this information helps others who have lost their hard-earned money due to misplaced trust. If you’re in a similar situation, you can contact them through their info below to seek help in recovering your stolen funds. 

Email: greatwhiprecoverycyberservices@gmail.com 
Website https://greatwhiprecoveryc.wixsite.com/greatwhip-site 
Phone Call:+1(406)2729101
![fkm](https://github.com/user-attachments/assets/ed5f5d47-3592-4e3b-8894-6fe505804a0f)

**Comment by keithwilson9899-stack** [comment]
ETHEREUM RECOVERY ASSISTANCE: CAPITAL CRYPTO RECOVER HELPED ME RECOVER $98,000 WORTH OF LOST ETH

In cases of cryptocurrency scams, having accurate information and trusted support is essential. I would like to recommend Capital Crypto Recover Service, a professional team that specializes in assisting individuals with the recovery of lost or stolen Bitcoin and Ethereum (ETH). Their experienced experts are dedicated to helping victims of digital asset fraud by carefully analyzing each case, developing strategic recovery plans, Capital Crypto Recover Service knowledgeable team's primary goals are to satisfy clients and offer significant support and working diligently toward fund retrieval. The team is committed to providing reliable assistance and maintaining a high level of client satisfaction. Based on my assessment, their reputation professionalism and a strong commitment to their clients. If you have experienced a cryptocurrency loss, you can contacting them for further assistance
 
Phone (Call/Text): +1 (336) 390-6684
Email: [Capitalcryptorecover@zohomail.com](mailto:Capitalcryptorecover@zohomail.com)
Alternate Email: [Recoverycapital@fastservice.com](mailto:Recoverycapital@fastservice.com)
Website: https://recovercapital.wixsite.com/capital-crypto-rec-1
![Uploading keith.png…]()

**Comment by keithwilson9899-stack** [comment]
ETHEREUM RECOVERY ASSISTANCE: CAPITAL CRYPTO RECOVER HELPED ME RECOVER $98,000 WORTH OF LOST ETH

In cases of cryptocurrency scams, having accurate information and trusted support is essential. I would like to recommend Capital Crypto Recover Service, a professional team that specializes in assisting individuals with the recovery of lost or stolen Bitcoin and Ethereum (ETH). Their experienced experts are dedicated to helping victims of digital asset fraud by carefully analyzing each case, developing strategic recovery plans, Capital Crypto Recover Service knowledgeable team's primary goals are to satisfy clients and offer significant support and working diligently toward fund retrieval. The team is committed to providing reliable assistance and maintaining a high level of client satisfaction. Based on my assessment, their reputation professionalism and a strong commitment to their clients. If you have experienced a cryptocurrency loss, you can contacting them for further assistance
 
Phone (Call/Text): +1 (336) 390-6684
Email: [Capitalcryptorecover@zohomail.com](mailto:Capitalcryptorecover@zohomail.com)
Alternate Email: [Recoverycapital@fastservice.com](mailto:Recoverycapital@fastservice.com)
Website: https://recovercapital.wixsite.com/capital-crypto-rec-1

**Comment by mayahenry087-commits** [comment]
I am not the first to recommend him because his recommendations can be seen everywhere but I am doing this because I am very happy, I am very happy because I made the right choice and that right choice is me chosen to follow my instincts and go with the good reviews i saw about hack digital spy, believe, if it's not for that choice i made i would have still been in a very toxic relationship with my ex partner who was a serial cheat but all that is gone now thanks to Mr. Hack digital spy everyone deserves happiness that includes you so to get that happiness you deserve I reckon. contact the best in the game for spying and gaining access into phone remotely without having the device on your hands contact email ; hackdigitalspy@gmail.com

**Comment by jacksonyamilet74-png** [comment]
The forensic report was the turning point Rating: ★★★★★ "When you lose $2M, you need a paper trail for the exchanges and legal teams. [Bitreclaim.com provided exactly that](https://bitreclaim.com). Their triangulation process is elite. After opening my case via email, they went to work mapping the transaction hashes. They found where the SGM scammers had parked the funds. Having a professional firm document the fraud makes all the difference when you're trying to get a freeze put on an account

**Comment by keithwilson9899-stack** [comment]
ETHEREUM RECOVERY ASSISTANCE: CAPITAL CRYPTO RECOVER HELPED ME RECOVER $98,000 WORTH OF LOST ETH

In cases of cryptocurrency scams, having accurate information and trusted support is essential. I would like to recommend Capital Crypto Recover Service, a professional team that specializes in assisting individuals with the recovery of lost or stolen Bitcoin and Ethereum (ETH). Their experienced experts are dedicated to helping victims of digital asset fraud by carefully analyzing each case, developing strategic recovery plans, Capital Crypto Recover Service knowledgeable team's primary goals are to satisfy clients and offer significant support and working diligently toward fund retrieval. The team is committed to providing reliable assistance and maintaining a high level of client satisfaction. Based on my assessment, their reputation professionalism and a strong commitment to their clients. If you have experienced a cryptocurrency loss, you can contacting them for further assistance
 
Phone (Call/Text): +1 (336) 390-6684
Email: [Capitalcryptorecover@zohomail.com](mailto:Capitalcryptorecover@zohomail.com)
Alternate Email: [Recoverycapital@fastservice.com](mailto:Recoverycapital@fastservice.com)
Website: https://recovercapital.wixsite.com/capital-crypto-rec-1
